docs/tutorials/load-test-grpc-multiple.md

```diff
@@ -0,0 +1,75 @@
+---
+template: main.html
+---
+
+# Load Test Multiple gRPC methods
+
+[Load Test gRPC with SLOs](./load-test-grpc.md) describes how to load test a single method from a gRPC service inside Kubernetes. This tutorial expands on the previous tutorial and describes how to load test multiple endpoints from an HTTP service.
+
+<p align='center'>
+  <img alt-text="load-test-grpc" src="../images/grpc.png" />
+</p>
+
+***
+
+???+ warning "Before you begin"
+    1. Try [your first experiment](../getting-started/your-first-experiment.md). Understand the main [concepts](../getting-started/concepts.md) behind Iter8 experiments.
+    2. Deploy the sample gRPC service in the Kubernetes cluster.
+    ```shell
+    kubectl create deployment routeguide  --image=golang --port=50051 \
+    -- bash -c "git clone -b v1.52.0 --depth 1 https://github.com/grpc/grpc-go; cd grpc-go/examples/route_guide; go run server/server.go"
+    kubectl expose deployment routeguide --port=50051
+    ```
+
+***
+
+## Launch experiment
+
+```shell
+iter8 k launch \
+--set "tasks={ready,grpc,assess}" \
+--set ready.deploy=routeguide \
+--set ready.service=routeguide \
+--set ready.timeout=60s \
+--set grpc.host=routeguide.default:50051 \
+--set grpc.endpoints.getFeature.call=routeguide.RouteGuide.GetFeature \
+--set grpc.endpoints.getFeature.dataURL=https://raw.githubusercontent.com/iter8-tools/docs/main/samples/grpc-payload/unary.json \
+--set grpc.endpoints.listFeature.call=routeguide.RouteGuide.ListFeatures \
+--set grpc.endpoints.listFeature.dataURL=https://raw.githubusercontent.com/iter8-tools/docs/main/samples/grpc-payload/server.json \
+--set grpc.protoURL=https://raw.githubusercontent.com/grpc/grpc-go/master/examples/route_guide/routeguide/route_guide.proto \
+--set assess.SLOs.upper.grpc/getFeature/error-rate=0 \
+--set assess.SLOs.upper.grpc/getFeature/latency/mean=50 \
```

Comment:
`provider-name[-endpoint-name]/metric-name[/aggregation]`

***

A few examples ... 

1. `grpc-getFeature/latency/mean` -- all 3 clauses.
2. `http/latency-mean` -- 2 out of 3 clauses.
3. `grpc/latency/mean` -- 2 out of 3 clauses.
4. `http-httpbin/latency-mean` - 2 out of 3 clauses.
5. `abn/sample_metric/count` - 3 out of 3 clauses.

Reply:
The above format for metrics needs to be explained in the user-guide under metrics topic. Appropriate explanations may need to accompany tutorials and task descriptions also.